frontend/src/components/donation-filters/donation.filters.jsx

```diff
@@ -0,0 +1,35 @@
+import { useCallback, useState } from "react";
+import { Input, Select } from "../../shared-components";
+import styles from "./donation.filters.module.scss";
+
+export function DonationFilters({ onChange }) {
+	const [filters, setFilters] = useState({
+		contains: null,
+		isActive: null,
+	});
+
+	const applyFilters = useCallback((newFilters) => {
+		const updatedFilters = { ...filters, ...newFilters };
+		setFilters(updatedFilters);
+		onChange(1, updatedFilters.contains, updatedFilters.isActive);
+	}, [filters, onChange]);
+
+	return (
+		<div className={styles.filters}>
+			<Input
+				withSearchIcon
+				id="donationContains"
+				name="donationContains"
+				placeholder="Pretraži..."
+				onChange={value => applyFilters({contains: value})}
+			/>
+			<Select
+				id="donationStatus"
+				name="donationStatus"
+				options={{ true: "u toku", false: "završeno" }}
+				placeholder="Status..."
+				onChange={value => applyFilters({isActive: value})}
+			/>
+		</div>
+	);
+}

```

Comment:
Ova komponenta ima gotovo isti kod kao blog.filters, sta mislis da napravimo to da bude reusable? Mozemo dodati kao tech debt, ne moramo sada.

Reply:
Da, svestan sam toga da su skoro isti, ali nisam zeleo da gubim puno vremena na abstrahovanje delova koji se ponavljaju, pogotovo ako se ponavljaju samo na 2 mesta i nece se vise ponavljati u projektu. Svakako mozemo to da uradimo ako nam bude ostalo vremana.

Reply:
Jasno, videcemo ako budemo imali vremena pa cemo uraditi u narednim iteracijama, resolvovacu ovaj komentar